CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        public JokeService JokeService { get => _jokeService; set => _jokeService = value ?? throw new ArgumentNullException(value.ToString()); }
+        private JokeService? _jokeService;
```

Comment:
```suggestion
        private IService? _JokeService;
```
You should be using the interface type `IService` instead of `JokeService`. Additionally, make sure private class fields are PascalCase. You'll need to modify all existing occurrences of `_jokeService` with `_JokeService`.